extensions/eclipse-che-theia-plugin-ext/src/browser/oauth-utils.ts

```diff
@@ -42,7 +42,7 @@ export class OauthUtils {
 
     async isAuthenticated(provider: string): Promise<boolean> {
         const token = await this.cheApiService.getOAuthToken(provider);
-        return token !== null;
+        return token !== undefined;
```

Comment:
@vinokurig is it ok to replace `!=== null` by undefined as reported by eslint ?

Reply:
It's doesn't matter because I am going to rework it in this PR: https://github.com/eclipse/che-theia/pull/694/files#diff-c08dd0360535189a619213d76179e0e5R71-R78